layers/+lang/go/packages.el

```diff
@@ -67,25 +68,25 @@
                                    "-check.f"
                                  "-run")))
               (save-excursion
-                  (re-search-backward "^func[ ]+\\(([[:alnum:]]*?[ ]?[*]?[[:alnum:]]+)[ ]+\\)?\\(Test[[:alnum:]_]+\\)(.*)")
```

Comment:
The change on this hunk is non-functional and probably accidental; it just de-indents the code by two spaces. This makes it look like you changed more code than you “really” did, which can be distracting.